### PR TITLE
operator: TestUpgrade: make expected*Image to follow ImageMinVersion

### LINKED ISSUE #1005: TestUpgrade: make expected*Image to follow ImageMinVersion

### PR DESCRIPTION
**Pull request by ozhuraki**
Closes #1005

### CODE REVIEW

File: pkg/controllers/reconciler_test.go [hunk 1]
```diff
@@ -27,31 +30,31 @@ func TestUpgrade(test *testing.T) {
...
 			upgrade:           true,
 		},
 		{
-			image:             "intel/intel-dsa-plugin:0.24.0",
-			expectedImage:     "intel/intel-dsa-plugin:0.24.0",
-			initimage:         "intel/intel-idxd-config-initcontainer:0.24.0",
-			expectedInitimage: "intel/intel-idxd-config-initcontainer:0.24.0",
+			image:             name + version,
+			expectedImage:     name + version,
+			initimage:         name + version,
```
Comment (bart0sh) [inline pkg/controllers/reconciler_test.go:42]: @ozhuraki can you explain why init image has the same name as plugin image?

**Comment by ozhuraki** [comment]
@bart0sh 
Updated, please take a look

**Review by bart0sh — APPROVED** [review]
/lgtm

**Review by mythi — APPROVED** [review]
thanks!

**Pull request merged**